### PR TITLE
Complete code coverage

### PR DESCRIPTION
**Pull request by simon111111**
This pull request fixes a couple issues and tests for code coverage.

### CODE REVIEW

File: tests/gaplint.test.py [hunk 1]
```diff
@@ -48,6 +53,34 @@ def test_orange_string(self):
...
+        gaplint._info_verbose('test/tests.g', 0,'msg')
+
+    def test_info_warn(self):
+        gaplint._SILENT = False
+        with self.assertRaises(AssertionError):
+            gaplint._info_warn(0, 0, 0, 0)
+        with self.assertRaises(AssertionError):
+            gaplint._info_warn('test', 'test', 'test', 'test')  
+        gaplint._info_warn('tests/test.g', 0, 'test', 0) 
+#______________________________________________________________________________
```
Comment (james-d-mitchell) [inline tests/gaplint.test.py]: Delete this line

File: tests/gaplint.test.py [hunk 2]
```diff
@@ -91,5 +126,77 @@ def test_ReplaceQuotes(self):
         self.assertEquals(ro.msg, 'invalid continuation of string')
         self.assertEquals(ro.abort, True)
 
+# skt4 addition
```
Comment (james-d-mitchell) [inline tests/gaplint.test.py]: And delete this

File: tests/gaplint.test.py [hunk 3]
```diff
@@ -91,5 +126,77 @@ def test_ReplaceQuotes(self):
...
+        ro = rule(', z; end;')
+        
+    def test_run_gaplint(self):     
+        with self.assertRaises(SystemExit):
+            run_gaplint()
+        with self.assertRaises(SystemExit):
+            run_gaplint(files=['tests/test.g'], max_warnings=0)        
+        run_gaplint(files=['non-existant-file'])
+        run_gaplint(files=['tests/test.g'], verbose=True)     
+#______________________________________________________________________________
```
Comment (james-d-mitchell) [inline tests/gaplint.test.py]: delete this line

**Comment by simon111111** [comment]
Minor changes completed:
- Trailing whitespace removed.
- Lines indicated have been removed.
- Added new test file, test3.g, and test for documents containing one character.

**Pull request merged**